tests/test_modeling_utils.py

```diff
@@ -300,6 +303,15 @@ def test_model_from_pretrained_with_different_pretrained_model_name(self):
                 BertModel.from_pretrained(TINY_T5)
         self.assertTrue("You are using a model of type t5 to instantiate a model of type bert" in cl.out)
 
+    @require_accelerate
+    def test_model_from_pretrained_with_none_quantization_config(self):
+        model = None
+        # Needs device_map for low_cpu_mem trigger & missing keys in base model load to trigger.
+        model = AutoModelForSequenceClassification.from_pretrained(
+            TINY_LLAMA, config=LlamaConfig.from_pretrained(TINY_LLAMA), device_map="auto", quantization_config=None
```

Comment:
```suggestion
            TINY_LLAMA, device_map="auto", quantization_config=None
```
Is passing the config really needed to repro? Otherwise you could use `AutoConfig`